generated/niscope/enums.py

```diff
@@ -561,68 +538,68 @@ class TriggerModifier(Enum):
 class TriggerSlope(Enum):
     NEGATIVE = 0
     '''
-    Specifies a falling edge (negative slope).
+    Falling edge
     '''
     POSITIVE = 1
     '''
-    Specifies a rising edge (positive slope).
+    Rising edge
     '''
 
 
 class TriggerType(Enum):
-    EDGE = 1
+    EDGE_TRIGGER = 1
```

Comment:
Here we have the same problem you fixed by looking at common prefixes, but with suffixes because that's how the NI-FGEN developers who added this roll.

`nifgen.TriggerType.TV_TRIGGER` instead of the more natural `nifgen.TriggerType.TV`

Should we look for common suffixes?

Reply:
I added a check for common suffixes too